core/src/sumeragi/network_topology.rs

```diff
@@ -424,9 +397,10 @@ impl Topology {
         &self.view_change_proofs
     }
 
-    /// Number of view changes.
-    pub const fn max_faults(&self) -> u32 {
-        self.max_faults
+    /// Maximum number of faulty peers that the network will tolerate.
+    pub fn max_faults(&self) -> usize {
+        #![allow(clippy::integer_division)]
```

Comment:
should you write a comment why this is allowed to be more in line with the contributing guide?